### ISSUE TITLE
If I entered 1.8V from the PCB to the 3.3V analog VDD, will it work?

### ISSUE THREAD

**Issue by Shehab-Naga** [efabless/caravel#490]
The Caravel has 4 power domains, 2 digital with 1.8 VDD and 2 analog with 3.3 VDD. In my project, I use four separate "digital" power domains, so I will need to use the analog domains for digital circuits.

My question is to able to use the analog domains for digital circuits, Can I just enter 1.8V from the PCB to the 3.3V analog VDD? or do I need to implement LDO to transfer the 3.3 VDD to 1.8 VDD?

**Comment by RTimothyEdwards** [comment]
Yes, you can just connect the VDDA1 and VDDA2 to the 1.8V lines.  These two power domains are unconnected on the development board by default, so you just need to wire them directly to one of the 3.3V header pins.

The VDDA1 and VDDA2 are flexible because they connect to absolutely nothing in the chip (excluding the user project) except for some ESD circuits.  The ESD circuits are power clamps rated for 3.3V to 5V operation, but they will operate (presumably with a bit less ESD protection) at 1.8V supply.
